test/test_componentdefinition.py

```diff
@@ -215,6 +215,17 @@ def test_hidden_sequence(self):
                                            rdflib.URIRef(sbol2.SBOL_SEQUENCE_PROPERTY),
                                            rdflib.URIRef(seq.identity)))
 
+    @unittest.expectedFailure
```

Comment:
Is this the test that demonstrates #272 ? If so, can a comment be added to point to issue 272?

Reply:
Done